site/en/docs/privacy-sandbox/user-agent/index.md

```diff
@@ -63,19 +67,19 @@ problematic because:
 *  the granularity and abundance of detail can lead to user identification;
 *  and the default availability of this information can lead to covert tracking.
 
```

Comment:
A quick style guide tip on using full sentences in lists:

https://developers.google.com/style/lists?hl=en

Reply:
And this edit also achieves parallelisation, which is always a good format for list items.